django/contrib/gis/db/backends/postgis/adapter.py

```diff
@@ -19,7 +19,12 @@ def __init__(self, obj, geography=False):
 
         # Getting the WKB (in string form, to allow easy pickling of
         # the adaptor) and the SRID from the geometry or raster.
-        if self.is_geometry:
+        if (isinstance(obj, PostGISAdapter) and not obj.ewkb or
+                isinstance(obj, Geometry) and obj.empty and obj.geom_type == 'Point'):
```

Comment:
In a brief spot check, I noticed that no tests are failing if `obj.geom_type == 'Point'` is removed. Not sure if it's something that should be tested or not.